platforms/java/com/area9innovation/flow/FlowRuntime.java

```diff
@@ -200,8 +203,12 @@ public static void toStringAppend(Object value, StringBuilder buf) {
 	}
 
 	public static String doubleToString(double value) {
-		String rstr = Double.toString(value);
-		return rstr.endsWith(".0") ? rstr.substring(0, rstr.length()-2) : rstr;
+		DecimalFormat df = new DecimalFormat("0");
```

Comment:
I suggest putting the DecimalFormat in an static member to avoid the "new" each time. That should make less "garbage".